src/DIRAC/Resources/Computing/SingularityComputingElement.py

```diff
@@ -255,20 +255,21 @@ def __createWorkArea(self, jobDesc=None, log=None, logLevel="INFO", proxy=None):
             self.log.warn("No user proxy")
 
         # Job Wrapper (Standard-ish DIRAC wrapper)
-        result = createRelocatedJobWrapper(
+        result = createJobWrapper(
             wrapperPath=tmpDir,
             rootLocation=self.__innerdir,
             jobID=jobDesc.get("jobID", 0),
             jobParams=jobDesc.get("jobParams", {}),
             resourceParams=jobDesc.get("resourceParams", {}),
             optimizerParams=jobDesc.get("optimizerParams", {}),
+            pythonPath="python",
             log=log,
             logLevel=logLevel,
             extraOptions="" if self.__installDIRACInContainer else "/tmp/pilot.cfg",
         )
         if not result["OK"]:
             return result
-        wrapperPath = result["Value"]
+        wrapperPath = result["Value"]["JobExecutableRelocatedPath"]
```

Comment:
`JobExecutableRelocatedPath` seems to be an optional entry in the dictionary? is it guaranteed that this is always present here?